src/pages/ActorPage/ActorPage.tsx

```diff
@@ -71,7 +71,10 @@ function ActorPage() {
             ?.slice(0, MAX_ACTOR_FILMS)
             .map(({ title, vote_average, poster_path, id, genre_ids }: TypeMovieCard) => {
               const genres = genre_ids
-                .map((genreId) => `${genresArray.find((el) => el.id === genreId)?.name} ` || '')
+                .map((genreId) => {
```

Comment:
Let's convert this calculation to a function and move this function to helpers.ts as it's repeated at least 3 time in different components.